tutor/templates/jobs/init/mounted-directories.sh

```diff
@@ -1,7 +1,23 @@
+# The initialization job contains various re-install operations needed to be done
+# on mounted directories (edx-platform, /mnt/*xblock)
+# 1. /mnt/*xblock
+# Whenever xblocks are mounted, during the image build time, they are copied over to container and
+# installed. This results in egg_info generation for the mounted xblocks. However, the egg_info is not carried
+# over to host. When the containers are launched, the host directories without egg_info are mounted on runtime
+# and disappear from pip list.
+# 2. edx-platform
 # When a new local copy of edx-platform is bind-mounted, certain build
 # artifacts from the openedx image's edx-platform directory are lost.
 # We regenerate them here.
 
+
+for mounted_xblock in /mnt/*xblock/; do
```

Comment:
We should not iterate just on xblocks, because we can mount other apps in this directory. See the list of auto-mounted directories here: https://github.com/overhangio/tutor/blob/f211b982fd8d970a538eb8452e16259a5eaae92b/tutor/plugins/openedx.py#L62

I think it should be safe to iterate on all directories that include a setup.py file.